docs/en/docs/network/config/multus-cr.md

```diff
@@ -1,41 +1,51 @@
+---
+MTPE: WANG0608GitHub
+DATE: 2024-08-01
+---
+
 # Creating a Multus CR
 
-Multus CR management is the secondary encapsulation of configuration instances in Multus CNI by Spiderpool. It aims to provide more flexible network connectivity and configuration options for containers to meet different network requirements and provide users with a simpler and more cost-effective experience. This page explains how to create a Multus CR before using multiple NIC configurations for creating workloads.
+Multus CR management is the secondary encapsulation of configuration instances in Multus CNI by Spiderpool.
+It aims to provide more flexible network connectivity and configuration options for containers
+to meet different network requirements and provide users with a simpler and more cost-effective experience.
+This page explains how to create a Multus CR before using multiple NIC configurations for creating workloads.
 If you need to create a new **Multus CR instance**, you can refer to this document.
 
-## Prerequisites
+## Prerequisite
 
-[SpiderPool successfully deployed](../modules/spiderpool/install/install.md), the new version of SpiderPool includes all the features of Multus-underlay.
+[SpiderPool is successfully deployed](../modules/spiderpool/install/install.md), and the new version of SpiderPool includes all the features of Multus-underlay.
```

Comment:
```suggestion
[SpiderPool has been successfully deployed](../modules/spiderpool/install/install.md) since [v0.8.7](https://github.com/spidernet-io/spiderpool/releases/tag/v0.8.7), which fully supports Multus-underlay.
```
中文也要相应修改，今天的`新`不是一直新，需要指出具体的版本号